doc/user_guide/checkers/features.rst

```diff
@@ -1198,7 +1198,8 @@ Refactoring checker Messages
   This message is emitted when pylint encounters boolean operation like"a < b
   and b < c", suggesting instead to refactor it to "a < b < c"
 :simplifiable-if-expression (R1719): *The if expression can be replaced with %s*
-  Used when an if expression can be replaced with 'bool(test)'.
+  Used when an if expression can be replaced with 'bool(test)' or simply 'test'
```

Comment:
This shouldn't really be in this PR. Not sure if we can make the documentation build as a pre-commit check in CI?

Reply:
We ruled against that at some point and decided to only upgrade with tbump for each release, but it does not make sense anymore as we added a lot of generated content and it would make sense to have to review the doc changes too for each code change. What do you think ?

Reply:
Yeah, I'm starting to wonder if adding the generated content was the right decision. It feels like it is giving us a lot of headaches at the moment. I would definitely be in favour of adding a `make html` step in the `pre-commit` CI run. But only in CI though, it would be very annoying to have to wait on the documentation build for every commit.

Reply:
Regarding the generated content I think it's the right decision, it's possible to modify a docstring on a checker that would break the doc if it's incompatible with RST, and if it's generated only at release we'll never catch it during review. Contributor can't know about every scripts that generates the documentation and the implication each change they do entice. Let me do something about this pre-commit step.

Reply:
#6736

Reply:
I just upgraded the generated content in #6775 because the generated content is unstable for some default value that depends on the OS or interpreter and #6736 is blocked.